ports/stm32/boards/NUCLEO_WB55/rfcore_debug.py

```diff
@@ -27,85 +27,16 @@
 # mechanism in the WB55, and works with the memory layout configured in
 # ports/stm32/rfcore.c -- i.e. it expects that rfcore_init() has been run.
 
-# At this stage this is useful for debugging, but can be extended to support
-# FUS/WS firmware updates.
```

Comment:
The comment here doesn't make much sense anymore, especially since this file is renamed to rfcore_debug.py.